themes/classic/templates/layouts/layout-both-columns.tpl

```diff
@@ -23,7 +23,7 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  *}
 <!doctype html>
-<html lang="{$language.iso_code}">
+<html lang="{$language.language_code|regex_replace:'/-.*/':''}">
```

Comment:
this should be handled at controller level

Reply:
You're right, fixed

Reply:
@PululuK, please approve if you agree.